src/utils.js

```diff
@@ -64,6 +64,21 @@ function isValidBarcode(value) {
   return ((10 - (result % 10)) % 10) === parseInt(paddedValue.charAt(13), 10)
 }
 
+function cleanBarcode(value) {
+  // remove spaces
+  value = value.replace(/\s/g, '')
```

Comment:
This shouldn't be required, the line below removes everything that is not a digit, including spaces.